charts/kube-prometheus-stack/hack/sync_prometheus_rules.py

```diff
@@ -29,49 +29,47 @@ def new_representer(dumper, data):
 
 refs = {
     # https://github.com/prometheus-operator/kube-prometheus
-    'ref.kube-prometheus': '76f2e1ef95be0df752037baa040781c5219e1fb3',
+    'ref.kube-prometheus': '407142589d5b38663e70f80c6c3b493e5b5e6578',
     # https://github.com/kubernetes-monitoring/kubernetes-mixin
-    'ref.kubernetes-mixin': '346bef2584068e803757e12c4ee4814e72a67927',
+    'ref.kubernetes-mixin': 'b247371d1780f530587a8d9dd04ccb19ea970ba0',
     # https://github.com/etcd-io/etcd
-    'ref.etcd': '84e67ffaf683cd4898897e958ba56da1f1bd2819',
+    'ref.etcd': '5a8c8b703b877702aef882662b941c2696247fec',
 }
 
 # Source files list
 charts = [
     {
-        'source': 'https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/%s/manifests/alertmanager-prometheusRule.yaml' % (refs['ref.kube-prometheus'],),
+        'git': 'https://github.com/prometheus-operator/kube-prometheus.git',
+        'branch': refs['ref.kube-prometheus'],
+        'source': 'main.libsonnet',
+        'cwd': 'jsonnet/kube-prometheus',
         'destination': '../templates/prometheus/rules-1.14',
```

Comment:
should we remove the -1.14 postfix?

Reply:
It feel like it make sense, maybe it's something for an follow-up PR?